test/hotspot/jtreg/testlibrary_tests/ir_framework/tests/TestBadFormat.java

```diff
@@ -843,12 +843,330 @@ public void mustSpecifyAtLeastOneConstraint2() {
+    public void applyIfCPUFeatureAndFaultyComparator() {}
+
+
+    @FailCount(2)
+    @Test
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "true"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "true", "avx"})
+    public void applyIfCPUFeatureOrNotEnoughFlags() {}
+
+    @FailCount(6)
+    @Test
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "true", "avx"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"Bla"})
+    public void applyIfCPUFeatureOrMissingValue() {}
+
+    @FailCount(5)
+    @Test
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"svee", "true", "sve", "< 34"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "!= 50", "Bla", "bla", "Bla2", "bla2"})
+    public void applyIfCPUFeatureOrUnknownFlag() {}
+
+    @FailCount(11)
+    @Test
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", ""})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "", "avx", ""})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"", "true"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"", "true", "", "true"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"", ""})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"", "", "", ""})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {" ", " ", " ", " "})
+    public void applyIfCPUFeatureOrEmptyValue() {}
+
+    @FailCount(20)
+    @Test
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "! 34"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "! 34", "avx", "! 34"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "!== 34"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "!== 34", "avx", "=== 34"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "<<= 34"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "<<= 34", "avx", ">>= 34"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "=<34"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "=<34", "avx", "=<34"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "<"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "<", "avx", "!="})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "!=0", "avx", "true"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", ">0", "avx", "true"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "<0", "avx", "true"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", ">=0", "avx", "true"})
+    @IR(failOn = IRNode.CALL, applyIfCPUFeatureOr = {"sve", "<=0", "avx", "true"})
+    public void applyIfCPUFeatureOrFaultyComparator() {}
+
+
     @FailCount(3)
     @Test
-    @IR(failOn = IRNode.CALL, applyIf = {"TLABRefillWasteFraction", "50"}, applyIfNot = {"UseTLAB", "true"})
-    @IR(failOn = IRNode.CALL, applyIfAnd = {"TLABRefillWasteFraction", "50", "UseTLAB", "true"},
```

Comment:
Not sure why you changed the format here. You seem to have one-liners everywhere else? Plus it just creates noise when reviewing.